screenpipe-app-tauri/lib/hooks/use-settings.tsx

```diff
@@ -102,7 +102,7 @@ const defaultSettings: Settings = {
   showScreenpipeShortcut: "Super+Alt+S",
   isFirstTimeUser: true,
   enableFrameCache: false, // Add this line
-  enableUiMonitoring: false, // Add this line
+  enableUiMonitoring: platform() === "macos", // Change this line to be macOS-specific
```

Comment:
does this compile?

usually you cant compile using `platform()` at buildtime you need to use at runtime